### PR TITLE
[11.0] [MIG] account fiscal position rule

### PR DESCRIPTION
**Pull request by hugosantosred**

### CODE REVIEW

**Review by mbcosta — APPROVED** [review]

File: account_fiscal_position_rule/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (Tonow-c2c) [inline account_fiscal_position_rule/__init__.py:1]: Remove This line [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration)

File: account_fiscal_position_rule/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```
Comment (Tonow-c2c) [inline account_fiscal_position_rule/__manifest__.py:1]: Remove This line [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration)

File: account_fiscal_position_rule/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+#   @author Renato Lima <renato.lima@akretion.com>
+# Copyright 2012-TODAY Camptocamp SA
+#   @author: Guewen Baconnier
+# Copyright 2018 - TODAY FactorLibre
+#   @author: Hugo Santos
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Account Fiscal Position Rule',
+    'version': '10.0.1.0.0',
```
Comment (Tonow-c2c) [inline account_fiscal_position_rule/__manifest__.py:13]: ```'version': '11.0.1.0.0',```

File: account_fiscal_position_rule/models/account_fiscal_position_rule.py [hunk 4]
```diff
@@ -0,0 +1,326 @@
+# -*- coding: utf-8 -*-
```
Comment (Tonow-c2c) [inline account_fiscal_position_rule/models/account_fiscal_position_rule.py:1]: Remove This line [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration)

File: account_fiscal_position_rule/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```
Comment (Tonow-c2c) [inline account_fiscal_position_rule/models/__init__.py:1]: Remove This line [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration)

File: account_fiscal_position_rule/models/account_invoice.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```
Comment (Tonow-c2c) [inline account_fiscal_position_rule/models/account_invoice.py:1]: Remove This line [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration)

File: account_fiscal_position_rule/README.rst [hunk 7]
```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (Tonow-c2c) [inline account_fiscal_position_rule/README.rst:1]: New Readme template in OCA:
* [https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)
* [https://github.com/OCA/maintainer-tools/tree/master/template/module/readme](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)

**Review by Tonow-c2c — CHANGES_REQUESTED** [review]
Check [https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0)

File: account_fiscal_position_rule/security/account_fiscal_position_rule_security.xml [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+<odoo>
+
+   <record id="account_fiscal_position_rule_rule" model="ir.rule">
+     <field name="name">Fiscal Position Rule</field>
+     <field name="model_id" ref="model_account_fiscal_position_rule"/>
+     <field name="global" eval="True"/>
+     <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+   </record>
+
+</odoo>
```
Comment (ghost) [inline account_fiscal_position_rule/security/account_fiscal_position_rule_security.xml:11]: Please use indentation `4 spaces`

File: account_fiscal_position_rule/test/account_fiscal_position_rule_data.xml [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+    <field name="chart_template_id" ref="l10n_generic_coa.configurable_chart_template"/>
+  </record>
+ 
+  <record id="fiscal_position_tax_exempt_template2" model="account.fiscal.position.template">
+    <field name="name">Tax Exempt</field>
+    <field name="chart_template_id" ref="l10n_generic_coa.configurable_chart_template"/>
+  </record>
+
+</odoo>
+
```
Comment (ghost) [inline account_fiscal_position_rule/test/account_fiscal_position_rule_data.xml:18]: Please use indentation `4 spaces`

File: account_fiscal_position_rule/test/test_rules.yml [hunk 10]
```diff
@@ -0,0 +1,107 @@
+-
```
Comment (ghost) [inline account_fiscal_position_rule/test/test_rules.yml:1]: If possible migrate to py.test

File: account_fiscal_position_rule/views/account_fiscal_position_rule_view.xml [hunk 11]
```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+        <!-- Create Menu for Configuration and Generation Templates -->
```
Comment (ghost) [inline account_fiscal_position_rule/views/account_fiscal_position_rule_view.xml:4]: Please use indentation `4 spaces`

**Review by ghost — COMMENTED** [review]
Some changes required

File: account_fiscal_position_rule/models/account_fiscal_position_rule.py [hunk 12]
```diff
@@ -0,0 +1,326 @@
...
+            'vat_rule': template.vat_rule,
+        }
+
+    @api.multi
+    def action_create(self):
+        obj_fpr_temp = self.env['account.fiscal.position.rule.template']
+        company_id = self.company_id.id
+
+        fsc_rule_template = obj_fpr_temp.search([])
+        # TODO fix me doesn't work multi template that have empty fiscal
```
Comment (leemannd) [inline account_fiscal_position_rule/models/account_fiscal_position_rule.py:314]: I'm not fond of improvement requests while migrating a module, but what's the current status of this TODO?
If it's still needed, do you plan to work on it?

**Review by leemannd — COMMENTED** [review]
LGTM. Only one question for the moment

**Pull request closed without merging**